### PR TITLE
Task/tam 2/pet name

### PR DESCRIPTION
**Pull request by kalibria**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,8 +8,9 @@
     "@testing-library/user-event": "^13.5.0",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-redux": "^7.2.6",
     "react-scripts": "5.0.0",
-    "web-vitals": "^2.1.4"
+    "redux": "^4.1.2"
```
Comment (ikushlianski) [inline package.json]: you could remove redux if it's not yet used

File: src/App.js [hunk 2]
```diff
@@ -1,25 +1,28 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import s from './App.module.css';
```
Comment (ikushlianski) [inline src/App.js]: what does `s` mean here?

Comment (ikushlianski) [reply src/App.js]: name should be more descriptive

File: src/App.js [hunk 3]
```diff
@@ -1,25 +1,28 @@
...
-          Learn React
-        </a>
-      </header>
+import EnterName from "./components/EnterName";
+
+
+
+const App = (props) => {
+    return <div className={s.mainField}>
+        <EnterName name={props.state.state.name}/>
```
Comment (ikushlianski) [inline src/App.js]: `props.state.state.name` looks too nested. Try to:
1) use object destructuring
2) reduce nesting so you don't have to use `state.state`

File: src/common/error.js [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const Error = () => {
+    return <div>
+        <p>Please, enter only Latin letter</p>
```
Comment (ikushlianski) [inline src/common/error.js]: should be `letters` instead of `letter`

File: src/components/EnterName.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+                <div>{state}</div>
+                <button onClick={() => {
+                    setEditMode(true)
+                }}>Edit name
+                </button>
+            </div>}
+        {error && <Error/>}
+
+
+    </div>
```
Comment (ikushlianski) [inline src/components/EnterName.js]: there should be no empty lines here. I think installing `prettier` would help. I will assist you with that

File: src/App.js (hunk 2 above)
Comment (kalibria) [reply src/App.js]: fixed

File: package.json (hunk 1 above)
Comment (kalibria) [reply package.json]: fixed

File: src/App.js (hunk 3 above)
Comment (kalibria) [reply src/App.js]: fixed

File: src/common/error.js (hunk 4 above)
Comment (kalibria) [reply src/common/error.js]: fixed

File: src/components/EnterName.js (hunk 5 above)
Comment (kalibria) [reply src/components/EnterName.js]: fixed

**Comment by ikushlianski** [comment]
Better split review fixes into several commits, each with its particular purpose (e.g. 1) Remove redux dependency 2) destructure props etc.) instead of having one commit "fixes after review" which does not describe what exactly changed

**Review by ikushlianski — APPROVED** [review]

**Review by ikushlianski — APPROVED** [review]

**Pull request merged**